### ISSUE TITLE
multi Topics implementation

### ISSUE THREAD

**Issue by khounish** [mmolimar/kafka-connect-fs#46]
Hi Molimar

I am doing a POC
Here is my setup. I am getting data from mysql->kafkconnect->potgresql->kafkaconnect->elasticsearch->kafkaconnect->hdfs->kafkaconnect->postgresql. 

I have 2 tables and thus 2 separate topics for each table.   i wanted to try something below ( bold mark)
name=FsSourceConnector
connector.class=com.github.mmolimar.kafka.connect.fs.FsSourceConnector
tasks.max=1
fs.uris=hdfs://localhost:9000//topics/eselastic-**poctabbackup/partition=0/,hdfs://localhost:9000//topics/eselastic-poctab1backup/partition=0/
topic=hdfs-eselastic-poctabbackup,hdfs-eselastic-poctabbackup1**
policy.class=com.github.mmolimar.kafka.connect.fs.policy.SimplePolicy
policy.recursive=true
policy.regexp=.*
file_reader.class=com.github.mmolimar.kafka.connect.fs.file.reader.AvroFileReader

In short, I wanted to check if multitopic can be controlled from the same configuration files like other connectors.. 
Example: HDFS SINC

name=hdfs-sink
connector.class=io.confluent.connect.hdfs.HdfsSinkConnector
tasks.max=1
topics=**eselastic-poctabbackup,eselastic-poctab1backup**
hdfs.url=hdfs://locahohost :9000/
flush.size=1

**Comment by khounish** [comment]
Closing the issue as I have found the workaround.
